smart-contract/assembly/contracts/utils/const.ts

```diff
@@ -0,0 +1,8 @@
+import { ConstantManager, KeyIncrementer } from '@massalabs/massa-as-sdk';
+const k = new KeyIncrementer();
+export const FILE_TAG = k.nextKey();
+export const CHUNK_TAG = k.nextKey();
+export const CHUNK_NB_TAG = k.nextKey();
+export const METADATA_TAG = k.nextKey();
+export const MAX_STORAGE_VALUE_SIZE = k.nextKey();
+export const FILES_PATH_LIST = new ConstantManager<Array<string>, u8>(k);
```

Comment:
Would be nice to have the key next to it like:

```suggestion
const k = new KeyIncrementer(); // 0 as u8
export const FILE_TAG = k.nextKey(); // 1 as u8
export const CHUNK_TAG = k.nextKey(); // 2 as u8
export const CHUNK_NB_TAG = k.nextKey(); // 3 as u8
export const METADATA_TAG = k.nextKey(); // 4 as u8
export const MAX_STORAGE_VALUE_SIZE = k.nextKey(); // 5 as u8
export const FILES_PATH_LIST = new ConstantManager<Array<string>, u8>(k); 
```

Would be a great help for the implementation in the server

Reply:
It's exactly what I did in my last commit